### PR TITLE
Update efficiency of the pipeline

### PR DESCRIPTION
**Pull request by nvnieuwk**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/CenterForMedicalGeneticsGhent/nf-cmgg-germline/tree/master/.github/CONTRIBUTING.md)
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: conf/modules.config [hunk 1]
```diff
@@ -141,6 +141,7 @@ process {
     */
 
     withName: "^.*:JOINT_GENOTYPING:MERGE_BEDS\$" {
+        ext.args = "-d 10000"
```
Comment (mvheetve) [inline conf/modules.config:144]: make param

Comment (nvnieuwk) [reply conf/modules.config]: done! `--merge_distance <integer>` with default of 10000

**Comment by nvnieuwk** [comment]
This can help: https://gatk.broadinstitute.org/hc/en-us/articles/360036448572-GenomicsDBImport#--genomicsdb-vcf-buffer-size

File: conf/modules.config [hunk 2]
```diff
@@ -154,6 +157,7 @@ process {
         time = { check_max( 16.h  * task.attempt, 'time' ) }
         ext.args = {
             [
+                "--genomicsdb-vcf-buffer-size 16384000",
```
Comment (matthdsm) [inline conf/modules.config:160]: how did you get this number?

File: modules/local/merge_beds.nf [hunk 3]
```diff
@@ -9,6 +9,7 @@ process MERGE_BEDS {
 
     input:
     tuple val(meta), path(bed, stageAs: "?/*")
+    path(fai)
```
Comment (matthdsm) [inline modules/local/merge_beds.nf:12]:
```suggestion
    tuple val(meta2), path(fai)
```

?

File: conf/modules.config (hunk 2 above)
Comment (nvnieuwk) [reply conf/modules.config:160]: A suggestion from a developer on a forum post [here](https://sites.google.com/a/broadinstitute.org/legacy-gatk-forum-discussions/2020-01-07-2019-07-10/24420-GenomicsDBImport-error-File2TileDBBinaryException) (the comment from 2019-09-10)

File: modules/local/merge_beds.nf (hunk 3 above)
Comment (nvnieuwk) [reply modules/local/merge_beds.nf:12]: Eh it's a local module. I don't have meta by default on my `fai` channel

**Review by matthdsm — APPROVED** [review]

**Pull request merged**